plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -910,6 +927,375 @@ public void testSettingChangeDataFeedEnabledProperty()
+        assertUpdate("CREATE TABLE " + tableName + " (page_url VARCHAR, domain VARCHAR, views INTEGER) WITH (change_data_feed_enabled = true)");
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url1', 'domain1', 1), ('url2', 'domain2', 2), ('url3', 'domain3', 3)", 3);
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url4', 'domain4', 4), ('url5', 'domain5', 2), ('url6', 'domain6', 6)", 3);
+
+        assertUpdate("UPDATE " + tableName + " SET page_url = 'url22' WHERE views = 2", 2);
+        MaterializedResult result = computeActual("SELECT * FROM TABLE(system.table_changes('test_schema', '" + tableName + "'))");
+        List<MaterializedRow> resultWithoutTimestamp = getCdfColumnsWithoutTimestamp(result);
+        assertThat(resultWithoutTimestamp).containsOnly(
+                row("url1", "domain1", 1, "insert", 1),
+                row("url2", "domain2", 2, "insert", 1),
+                row("url3", "domain3", 3, "insert", 1),
+                row("url4", "domain4", 4, "insert", 2),
+                row("url5", "domain5", 2, "insert", 2),
+                row("url6", "domain6", 6, "insert", 2),
+                row("url2", "domain2", 2, "update_preimage", 3),
+                row("url22", "domain2", 2, "update_postimage", 3),
+                row("url5", "domain5", 2, "update_preimage", 3),
+                row("url22", "domain5", 2, "update_postimage", 3));
+
+        assertUpdate("DELETE FROM " + tableName + " WHERE views = 2", 2);
+        result = computeActual("SELECT * FROM TABLE(system.table_changes('test_schema', '" + tableName + "', 3))");
+        resultWithoutTimestamp = getCdfColumnsWithoutTimestamp(result);
+        assertThat(resultWithoutTimestamp).containsOnly(
+                row("url22", "domain2", 2, "delete", 4),
+                row("url22", "domain5", 2, "delete", 4));
+
+        result = computeActual("SELECT * FROM TABLE(system.table_changes('test_schema', '" + tableName + "')) ORDER BY _commit_version, _change_type, domain");
+        resultWithoutTimestamp = getCdfColumnsWithoutTimestamp(result);
+        assertThat(resultWithoutTimestamp).containsExactly(
+                row("url1", "domain1", 1, "insert", 1),
+                row("url2", "domain2", 2, "insert", 1),
+                row("url3", "domain3", 3, "insert", 1),
+                row("url4", "domain4", 4, "insert", 2),
+                row("url5", "domain5", 2, "insert", 2),
+                row("url6", "domain6", 6, "insert", 2),
+                row("url22", "domain2", 2, "update_postimage", 3),
+                row("url22", "domain5", 2, "update_postimage", 3),
+                row("url2", "domain2", 2, "update_preimage", 3),
+                row("url5", "domain5", 2, "update_preimage", 3),
+                row("url22", "domain2", 2, "delete", 4),
+                row("url22", "domain5", 2, "delete", 4));
+    }
+
+    @Test
+    public void testReadingChangesMadeByBasicOperationsOnPartitionedTableWithCdfEnabled()
```

Comment:
if is better if test names are informative.
for that, they need not to be long

`testReadChangesOnPartitionedTable`?
`testReadCdfOnPartitionedTable`?